simulator/kernel/mars/mars_kernel.cpp

```diff
@@ -6,8 +6,10 @@
 
 #include "mars_kernel.h"
 
+#include <infra/macro.h>
 #include <kernel/base_kernel.h>
 #include <memory/elf/elf_loader.h>
+#include <infra/exception.h>
```

Comment:
Please don't forget about lexicographical order.
The motivation is to reduce fiasco cases when something is dependent on specific order of includes. Alphabet randomizes the logic, but makes it more readable.